site/content/blog/simplifying-package-authoring.md

```diff
@@ -0,0 +1,301 @@
+
+Now, we have created our package and packageMetadata. 
+
+Let's see how the package and packageMetadata files look like:
+
+```bash
+$ cat carvel-artifacts/packages/dynatrace.carvel.dev/package.yml
+apiVersion: data.packaging.carvel.dev/v1alpha1
+kind: Package
+metadata:
+  creationTimestamp: null
+  name: dynatrace.carvel.dev.1.0.0
+spec:
+  refName: dynatrace.carvel.dev
+  releasedAt: "2022-08-15T11:54:50Z"
+  template:
+    spec:
+      deploy:
+      - kapp: {}
+      fetch:
+      - imgpkgBundle:
+          image: index.docker.io/rohitagg2020/dynatrace-bundle@sha256:d3fd67881ccba75134061451348130577ad0b2034d0a65b44b965b56e7d1c939
+      template:
+      - ytt:
+          paths:
+          - upstream
+      - kbld:
+          paths:
+          - '-'
+          - .imgpkg/images.yml
+  valuesSchema:
+    openAPIv3:
+      default: null
+      nullable: true
+  version: 1.0.0
+
+$ cat carvel-artifacts/packages/dynatrace.carvel.dev/metadata.yml
+apiVersion: data.packaging.carvel.dev/v1alpha1
+kind: PackageMetadata
+metadata:
+  creationTimestamp: null
+  name: dynatrace.carvel.dev
+spec:
+  displayName: dynatrace
+  longDescription: dynatrace.carvel.dev
+  shortDescription: dynatrace.carvel.dev
+```
+
+Next step is to add it to package repository. 
+
+### kctrl pkg repo release
+
+`kctrl` can be used to release packages grouped together as a `PackageRepository`. Let’s bundle the `dynatrace` package created above into the PackageRepository.
+
+Let's create a folder for our repository.
+```bash
+$ cd ../
+$ mkdir pkg-repo && cd pkg-repo
+$ mkdir packages && cd packages
+```
+
+`kctrl` will create a repo bundle from all the packages and metadata present in the `packages` folder. 
+We will copy the `package.yml` and `metadata.yml` created above into the `packages` folder. We will follow the bundle format as mentioned [here](). Alternatively, while running `pkg release`, `--repo-output` flag can be used to copy the `package.yml` and `metadata.yml` in the prescribed PackageRepository bundle format at a specified location.
+
+```bash
+$ mkdir dynatrace.carvel.dev && cd dynatrace.carvel.dev
+$ cp ../../../dynatrace/carvel-artifacts/packages/dynatrace.carvel.dev/package.yml 1.0.0.yml
+$ cp ../../../dynatrace/carvel-artifacts/packages/dynatrace.carvel.dev/metadata.yml .
+$ cd ../..
+$ tree
+.
+└── packages
+    └── dynatrace.carvel.dev
+        ├── 1.0.0.yml
+        └── metadata.yml
+```
+
+Now, we will run `kctrl pkg repo release` to create and release the repository bundle. This repository bundle can be distributed to the the Package Consumer.
+
+```bash
+$ kctrl pkg repo release -v 1.0.0
+```
+
+In this command, it will ask for package repository name and only 1 question about where to push our repository bundle. Also, we are versioning our package repository with `-v` flag.
```

Comment:
```suggestion
In this command, it will ask for package repository name and only one question about where to push our repository bundle. Also, we are versioning our package repository with `-v` flag.
```

Reply:
Adopted